.github/workflows/windows-nv-clang-d3d12.yaml

```diff
@@ -6,16 +6,16 @@ permissions:
 
 on:
   workflow_dispatch:
-  schedule:
-    - cron: '0 0-12 * * 1-5' # run every 30 minutes overnight
-    - cron: '0 * * * 0,6' # run every 30 minutes on weekends
+#  schedule:
+#    - cron: '0 0-12 * * 1-5' # run every 30 minutes overnight
+#    - cron: '0 * * * 0,6' # run every 30 minutes on weekends
```

Comment:
nit: Can we just delete this? Can use git history to bring it back if we end up needing it